phial/wrappers.py

```diff
@@ -143,3 +152,66 @@ def __init__(self,
 
     def __repr__(self) -> str:
         return "<Attachment in {0} >".format(self.channel)
+
+class MessageAttachment():
+    def __init__(self,
+                 fallback: Union[str, None] = None,
```

Comment:
Could you please switch these to use `Optional[str]` instead of a `Union` with none? (Sorry know this ones my fault just fixed in #34 )